### PR TITLE
[MDS-5844] Declaration page

### PR DESCRIPTION
**Pull request by taraepp**
## Objective 
- add the declaration page

[MDS-5844](https://bcmines.atlassian.net/browse/MDS-5844)

_Why are you making this change? Provide a short explanation and/or screenshots_
- gif of page fold wrapper component in action is too big for GH, but it's posted on the ticket

### CODE REVIEW

File: services/common/src/redux/selectors/projectSelectors.ts [hunk 1]
```diff
@@ -39,26 +39,31 @@ const formatProjectContact = (contacts): IProjectContact[] => {
 
   return formattedContacts;
 };
-const formatAuthorizations = (authorizations = [], authTypes) => {
+const formatAuthorizations = (authorizations = [], amsAuthTypes, statusCode) => {
```
Comment (taraepp) [inline services/common/src/redux/selectors/projectSelectors.ts:42]: this param was always amsAuthTypes, just made it more obvious

File: services/minespace-web/src/components/Forms/projects/informationRequirementsTable/ReviewSubmitInformationRequirementsTable.js [hunk 2]
```diff
@@ -10,11 +10,15 @@ const propTypes = {
   sub_requirement: customPropTypes.subRequirements.isRequired,
 };
 
-const renderSubrequirement = (item) => item.map((it) => <Subrequirement sub_requirement={it} />);
+const renderSubrequirement = (item) =>
+  item.map((it) => {
+    //eslint-disable-next-line @typescript-eslint/no-use-before-define
```
Comment (taraepp) [inline services/minespace-web/src/components/Forms/projects/informationRequirementsTable/ReviewSubmitInformationRequirementsTable.js:15]: all I wanted to do in this file was add a bordered=false param below, but eslint was not happy about that rule, and I found that the logic was kind of circular. I _did_ give it its missing key though.

File: services/minespace-web/src/styles/overrides/antd-overrides.scss [hunk 3]
```diff
@@ -280,7 +280,6 @@ h5.ant-typography,
 }
 
 .ant-table-thead>tr>th {
-  background: #F7F8FA;
```
Comment (taraepp) [inline services/minespace-web/src/styles/overrides/antd-overrides.scss:283]: This hasn't actually changed, but now it's in the webpack variables! Who knew that the index.pure.less rules I'm always fighting were defined by us all along...

File: services/minespace-web/src/styles/settings/theme.scss [hunk 4]
```diff
@@ -45,25 +45,17 @@ $black: #000;
...
 // alert info backgrounds.
-$primary-1: tint(
-  $primary-color,
-  90%
-); //color(~`colorPalette('${primary-color}', 1) `); // replace tint($primary-color, 90%)
-$primary-2: tint(
-  $primary-color,
-  80%
-); //color(~`colorPalette('${primary-color}', 2) `); // replace tint($primary-color, 80%)
+$primary-1: tint($primary-color,
```
Comment (taraepp) [inline services/minespace-web/src/styles/settings/theme.scss:48]: I think that all of the changes I made in `theme.scss` I ended up reverting in a later commit, so this is all eslint.

File: services/minespace-web/webpack.parts.js [hunk 5]
```diff
@@ -163,13 +163,13 @@ exports.loadCSS = ({ include, exclude } = {}) => ({
                   "border-color-base": "#003366",
                   "border-color-split": "#efefef",
                   "border-width-base": "2px",
-                  "background-color-light": "#f7f8fa",
+                  "background-color-light": "#f2f2f2",
```
Comment (taraepp) [inline services/minespace-web/webpack.parts.js:166]: Turns out that this is where we go for changing global style variables, I feel kind of silly for not knowing/remembering before. I see you, `border-radius: 5px`. 👀

File: services/minespace-web/webpack.config.ts [hunk 6]
```diff
@@ -215,7 +214,7 @@ const prodConfig = merge([
   }),
   parts.bundleOptimization({
     options: {
-      maxSize: 3000000,
+      // maxSize: 3000000,
```
Comment (taraepp) [inline services/minespace-web/webpack.config.ts:217]: Note for the future: the bundle splitting was causing errors in loading, follow up ticket made.

File: services/minespace-web/src/styles/components/Forms.scss [hunk 7]
```diff
@@ -13,6 +13,15 @@
...
 }
 
+// .ant-form-item-checkbox-required .ant-checkbox+span {
+//     &::after {
+//         content: "*";
+//         font-family: $font-family;
+//         color: $gov-red;
+//     }
+
+// }
```
Comment (henryoforeh-dev) [inline services/minespace-web/src/styles/components/Forms.scss]: Will this commented code still be used in future ?

**Review by henryoforeh-dev — DISMISSED** [review]
Nice, just a minor comment.

File: services/minespace-web/src/styles/components/Forms.scss (hunk 7 above)
Comment (taraepp) [reply services/minespace-web/src/styles/components/Forms.scss]: ah this got missed in my reverting/unreverting. Going to re-instate it- it's necessary to show required mark on checkbox. Good catch!

**Review by henryoforeh-dev — APPROVED** [review]

**Review by asinn134 — APPROVED** [review]

**Pull request merged**